pkg/machines/libvirt-dbus.js

```diff
@@ -555,6 +555,13 @@ LIBVIRT_DBUS_PROVIDER = {
                         props.id = objPath;
                         props.connectionName = connectionName;
 
+                        return call(connectionName, objPath, 'org.libvirt.Network', 'GetXMLDesc', [0], TIMEOUT);
+                    })
+                    .then(networkXml => {
+                        let result = xml2js(networkXml, { compact: true });
+
+                        props.xml = result;
```

Comment:
It feels like this commit ("add network resource XML to the store object") should be part of another commit that actually makes use of it.

Reply:
Makes sense. Simon introduced network details already in his Networks PR that was merged last week, so with rebase I will have them ready for use.